src/tables/sortable-table.js

```diff
@@ -47,39 +51,43 @@ const propTypes = {
   onChange: PropTypes.func,
   rowComponent: Types.component,
   headerComponent: Types.component,
-}
+  ...Caption.propTypes,
+  caption: PropTypes.string,
+};
 const defaultProps = {
-  className: '',
+  className: "",
   data: [],
   initialAscending: true,
-  initialColumn: '',
+  initialColumn: "",
   disableReverse: false,
   disableSort: false,
   controlled: false,
   onChange: noop,
-}
+  caption: "",
+};
 const defaultControls = {
-  initialSortPath: '',
+  initialSortPath: "",
   initialSortFunc: null,
   initialValueGetter: null,
-}
+};
 
 function getInitialSortControls(initialColumn, columns) {
-  if (!initialColumn) return defaultControls
+  if (!initialColumn) return defaultControls;
```

Comment:
It would be great if you could undo the formatting applied. I think your global prettier settings must be different than LaunchPad's